cmd/renterd/main.go

```diff
@@ -193,6 +193,7 @@ func main() {
 	flag.DurationVar(&autopilotCfg.AccountsRefillInterval, "autopilot.accountRefillInterval", defaultAccountRefillInterval, "interval at which the autopilot checks the workers' accounts balance and refills them if necessary")
 	flag.DurationVar(&autopilotCfg.Heartbeat, "autopilot.heartbeat", 30*time.Minute, "interval at which autopilot loop runs")
 	flag.Float64Var(&autopilotCfg.MigrationHealthCutoff, "autopilot.migrationHealthCutoff", 0.75, "health threshold below which slabs are migrated to new hosts")
+	flag.DurationVar(&autopilotCfg.RevisionBroadcastInterval, "autopilot.revisionBroadcastInterval", 24*time.Hour, "interval at which the autopilot broadcasts contract revisions to be mined - can be overwritten using the REVISION_BROADCAST_INTERVAL environment variable - setting it to 0 will disable this feature")
```

Comment:
I know it's quite lengthy but shouldn't we be prefixing env vars with meaningful prefixes? e.g. `RENTERD_AUTOPILOT_REVISION_BROADCAST_INTERVAL` is better right? Properly namespacing env variables is very useful.

Reply:
`nit` I would've probably also housed `RENTERD_MIGRATOR_PARALLEL_SLABS_PER_WORKER` under`RENTERD_AUTOPILOT_ etc ...`